docs/topics/auth/customizing.txt

```diff
@@ -190,11 +196,12 @@ exception in :meth:`~django.contrib.auth.models.User.has_perm()` or
 :meth:`~django.contrib.auth.models.User.has_module_perms()`, the authorization
 will immediately fail and Django won't check the backends that follow.
 
-The simple backend above could implement permissions for the magic admin
-fairly simply::
+A simple backend could implement permissions for the magic admin fairly
+simply::
```

Comment:
Since we're touching this sentence, do you think it worth dropping/changing one or other of the `simple` cognates?